### ISSUE TITLE
Improve handling of info message about unidriectional connections

### ISSUE THREAD

**Issue by adisos** [np-guard/vpc-network-config-analyzer#129]
see comments from:

https://github.com/np-guard/vpc-network-config-analyzer/pull/125#discussion_r1285416364 

few options to consider: 
- the info message should / should not be always added to the output
- the info message should be added once, not from various code functions

current state: info msg is `"connections are stateful unless marked with *"`, and is always added to textual output (txt,md)

- [x] #292
- [x] #294
- [x] Avoid involvement in unit tests expected output ?

Make sure that the info message will be added once, not from various code functions

consider also the comment from another PR : https://github.com/np-guard/vpc-network-config-analyzer/pull/207#discussion_r1378485943 
The report of aggregated VPCs should contain this line only once rather than after every VPC report.

**Comment by adisos** [comment]
- [ ] The report of aggregated VPCs should contain this line only once rather than after every VPC report.
- [ ] The info message should not be always added to the output
- [ ] Avoid involvement in unit tests expected output ?
- [ ] the info message should be added once, not from various code functions
